### PR TITLE
BW-1235 Update `terra-jupyter-base` and its child images (adds `workspace_cromwell.py` script in base image)

### PR DESCRIPTION
**Pull request by salonishah11**
Jenkins job [leonardo-custom-image-rtitle #164](https://fc-jenkins.dsp-techops.broadinstitute.org/view/Leonardo/job/leonardo-custom-image-rtitle/164/) failed, but it was able to create custom dataproc and GCE images successfully:

- [leo-524-gce-cos-image-c9bf4b6](https://console.cloud.google.com/compute/imagesDetail/projects/broad-dsp-gcr-public/global/images/leo-524-gce-cos-image-c9bf4b6?project=broad-dsp-gcr-public)
- [leo-524-image-dataproc-2-0-39-debian10-c9bf4b6](https://console.cloud.google.com/compute/imagesDetail/projects/broad-dsp-gcr-public/global/images/leo-524-image-dataproc-2-0-39-debian10-c9bf4b6?project=broad-dsp-gcr-public)

terra-docker PR: [#324](https://github.com/DataBiosphere/terra-docker/pull/324)

Closes https://broadworkbench.atlassian.net/browse/BW-1235

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green (you can re-run automation tests with a comment saying `jenkins retest`
- [ ] Run the automation tests multiple times in parallel to weed out instability if applicable via a comment saying `multi-test`
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by cjllanwarne — APPROVED** [review]

**Review by Qi77Qi — APPROVED** [review]
tha change itself looks good...but can this wait? we currenlty have some issue if merging this PR, terra ui integration might break...if we can wait until Fri to merge this, that'd be great

**Comment by salonishah11** [comment]
re: merging on Friday - [Talked on Slack](https://broadinstitute.slack.com/archives/CA3NP1733/p1653495344411489) and I will merge this tomorrow (as Friday is a holiday) after checking in with Justin.

**Pull request merged**
